keyboards/honeycomb/rules.mk

```diff
@@ -0,0 +1,82 @@
+
+OPT_DEFS += -Dhoneycomb_PROMICRO
+OPT_DEFS += -DCATERINA_BOOTLOADER
+honeycomb_UPLOAD_COMMAND = while [ ! -r $(USB) ]; do sleep 1; done; \
```

Comment:
This shouldn't be necessary.

You should be able to just run:
```bash
make honeycomb:default:avrdude
```

And this will handle flashing the controller, including COM port detection.